scu-de/src/fiskaltrust.Middleware.SCU.DE.DeutscheFiskal/DeutscheFiskalSCUConfiguration.cs

```diff
@@ -14,9 +14,9 @@ public class DeutscheFiskalSCUConfiguration
         public int MaxFccDownloadTimeSec { get; set; } = 60 * 60;
         public int ProcessTimeoutSec { get; set; } = 5 * 60;
         public string FccUri { get; set; }
-        public virtual string CertificationId { get; set; } = "BSI-K-TR-0474-2021";
+        public virtual string CertificationId { get; set; } = "BSI-K-TR-0524-2024";
         public bool DisplayCertificationIdAddition { get; set; } = true;
-        public string CertificationIdAddition { get; set; } = "USK ausgesetzt";
+        public string CertificationIdAddition { get; set; }
```

Comment:
IMO not a good idea to remove this, as it's sheer impossible to fulfill the Umgebungsschutz for this TSE.